### PR TITLE
Team B Wiki Entry - Fall 2020

### PR DESCRIPTION
**Pull request by brysonjones**
This commit has two entries added: 
- A* Planning Implementation Guide
- RTK GPS

### CODE REVIEW

File: _data/navigation.yml [hunk 1]
```diff
@@ -82,6 +82,8 @@ wiki:
         url: /wiki/sensing/opencv-stereo/
       - title: Camera-IMU Calibration using kalibr
         url: /wiki/sensing/camera-imu-calibration/
+      - title: RTK GPS
+	url: /wiki/sensing/gps/
```
Comment (smbryan) [inline _data/navigation.yml:86]: Double check the white space here. Should be spaces and the 'u' of url should line up with the 't' of title.

File: _data/navigation.yml [hunk 2]
```diff
@@ -250,6 +252,8 @@ wiki:
     children:
       - title: Planning Overview
         url: /wiki/planning/planning-overview/
+      - title: A* Planner Implementation Guide
+	url: /wiki/planning/astar_planning_implementation_guide/
```
Comment (smbryan) [inline _data/navigation.yml:256]: White space.

**Review by smbryan — CHANGES_REQUESTED** [review]
Solid submission. There are two lines in the navigation file that need attention. Will be ready to merge once those are fixed.

**Comment by brysonjones** [comment]
Just added the changes!

File: wiki/planning/astar_planning_implementation_guide.md [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+- g(s) value - cost of the shortest path from the start state (sstart) to the current state (s) so far
+- h(s) value - estimate of the cost-to-go from the current state (s) to the goal state (sgoal)
+- f(s) value - total estimated cost from the start state (sstart) to the goal state (sgoal)
+- Admissibility - h(s) is an underestimate of the true cost to goal
+- Monotonicity/Consistent - h(s) <= c(s,s’) + h(s’) for all successors s’ of s
+- Optimality - no path exists from the start to the goal with a lower cost within the constraints of the problem
+
+A* works by computing optimal g-values for all states along the search at any point in time.
+
+![](/assets/astar_h_viz.png)
```
Comment (smbryan) [inline wiki/planning/astar_planning_implementation_guide.md:32]: All asset paths should be relative -> `assets/astar_h_viz.png`.

Drop the first `/` and you'll be good. Please make sure you get them all.

**Review by smbryan — CHANGES_REQUESTED** [review]
Almost! With the corrected directories, we need to change the path to the file.

**Comment by brysonjones** [comment]
Just updated. Let me know if there are any other issues!

**Review by smbryan — APPROVED** [review]

**Comment by smbryan** [comment]
All set! Looking forward to checking out yinz posters soon.

**Pull request merged**